modules/secure-logging/tests/test_secure_logging.c

```diff
@@ -701,6 +783,137 @@ void test_slog_performance(void)
   closure(testData);
 }
 
+
+#define RUN_CLI_SLOG_SMOKETEST 0
+
+/* Notes in regard RUN_CLI_SLOG_SMOKETEST
+   *
+   * - For the time being, RUN_CLI_SLOG_SMOKETEST is set to 0 to disable
+   *   the test of cli tools in context of unit tests.
+   *   Developers might set it to 1 manually here at least on GNU Linux systems.
+   *   When RUN_CLI_SLOG_SMOKETEST is set to 1, unit test will last longer,
+   *   approximately two minutes instead of some seconds.
+   * - Currently the CI test do fail when set to 1. Reasons:
+   *   -- chmod is used to make test scripts and helper scripts executable
+   *      this causes in CI system a 'high severity security vulnerability'
+   *   -- Expected 'sed' is not available for CI macOS version, but used
+   *      by update_conf_path.sh.
+   */
+
+
+#if defined(RUN_CLI_SLOG_SMOKETEST) && ((RUN_CLI_SLOG_SMOKETEST) == 1)
+
+// Define the required permission constant:
+// S_IRWXU | S_IRGRP | S_IXGRP | S_IROTH | S_IXOTH
+// This translates to 755 (rwxr-xr-x)
+// #define SCRIPT_PERMISSIONS (S_IRWXU | S_IRGRP | S_IXGRP | S_IROTH | S_IXOTH)
+
+
+int make_executable(const char *filepath)
+{
+  struct stat st;
+  if (stat(filepath, &st) == -1)
+    {
+      cr_log_warn("make_executable, stat fails for %s\n", filepath);
+      msg_warning("make_executable, stat fails for ", evt_tag_str("file", filepath));
+      return -1; //-- ERROR
+    }
+  return 0; //-- only used to check if path is valid currently
+
+  // chmod is causing 'high severity security vulnerability' on GitHubs CI.
```

Comment:
this part is not needed anymore, i guess

Reply:
i cannot really see the difference, git should handle this on windows either
apart from that, probably a separate function can be used then instead, and in the function the platform differences can be handle better